src/SelectPanel/SelectPanel.tsx

```diff
@@ -0,0 +1,107 @@
+import React, {useCallback, useMemo} from 'react'
+
+import {FilteredActionList, FilteredActionListProps} from '../FilteredActionList'
+import {OverlayProps} from '../Overlay'
+import {ItemInput} from '../ActionList/List'
+import {FocusZoneSettings} from '../behaviors/focusZone'
+import {DropdownButton, DropdownButtonProps} from '../DropdownMenu'
+import {ItemProps} from '../ActionList'
+import {AnchoredOverlay} from '../AnchoredOverlay'
+import Flex from '../Flex'
+
+export interface SelectPanelProps extends Omit<FilteredActionListProps, 'onChange'> {
+  open: boolean
+  onOpen: (gesture: 'anchor-click' | 'anchor-key-press') => unknown
+  onClose: (gesture: 'click-outside' | 'escape') => unknown
```

Comment:
`ActionMenu` [uses `open` and `setOpen`](https://github.com/primer/components/blob/84c7e771f8ac9bb707fbb977658e9e40620bc20c/src/ActionMenu.tsx#L28-L36)—should we be consistent (either changing `ActionMenu` to use `open`/`onOpen`/`onClose`, or changing `SelectPanel` to use `open`/`setOpen`)?

Reply:
Going with the `setOpen` approach, passing `setOpen(open: boolean, gesture: 'possible gestures here')`.  We will need to circle back and probably change the `DropdownMenu` and `AnchoredOverlay` to have the same API

Reply:
> Going with the setOpen approach, passing setOpen(open: boolean, gesture: 'possible gestures here').

👍 Sounds good, thanks!

> We will need to circle back and probably change the `DropdownMenu` and `AnchoredOverlay` to have the same API.

Great idea! Could you please open an issue to track API consistency updates?